.travis.yml

```
@@ -1,26 +1,20 @@
 language: python
 python:
 - 3.6
-cache:
-  pip: true
-env:
-  global:
-  - PACKAGE_NAME=wagtail-draftail-snippet
-  - PIP_INDEX_URL=$PYPI_URL+simple/
+- 3.7
+- 3.8
+cache: pip
```

```suggestion
cache:
  pip: true
```
I think mine works, but this is more clear and offers the understanding that there are other potentially useful cache directives.